package.json

```diff
@@ -44,7 +44,7 @@
   "dependencies": {
     "@appium/base-driver": "^9.1.0",
     "@appium/support": "^5.1.1",
-    "@xmldom/xmldom": "^0.x",
+    "@xmldom/xmldom": "^0.9.2",
```

Comment:
not sure we need this as such pattern only includes patches for version 0.9

Reply:
yea. hm, at least this won't hurt smaller versions so let me keep x here